Makefile

```diff
@@ -43,6 +43,10 @@ deploy-appbundle:
 		$(BUILD_DIR)/alwaysontop.min.js \
 		$(BUILD_DIR)/alwaysontop.min.map \
 		$(OUTPUT_DIR)/analytics-ga.js \
+		$(BUILD_DIR)/analytics-ga.min.js \
```

Comment:
is `$(OUTPUT_DIR)/analytics-ga.js \` still needed?

Reply:
@saghul Discussed this with @aaronkvanmeerten and we agreed this will be the painless way for deploying the changes for first time. Once we deploy we'll remove it!